doc/architecture/decisions/0036-use-audited-for-budget-revisions.md

```diff
@@ -0,0 +1,53 @@
+# 36. Use Audited for Budget Revisions
+
+Date: 2023-03-15
+
+## Status
+
+Accepted
+
+## Context
+
+Budgets can go through a number of revisions; they can be updated over time as
+more or less funding comes in, or the scope of the activity changes.
+
+We want to be able to keep a history of these revisions. They will be displayed
+on a new revisions page and exported as "current" and "original" budgets.
+
+### Audited
+
+The [Audited Gem](https://github.com/collectiveidea/audited) provides us with
+an easy way to achieve the desired functionality, with minimal changes to the
+existing budget model.
+
+The features of Audited include:
+- the ability to keep an audit history, including limiting what is considered a
+  change
+- the ability to optionally attach a comment to an audit
+- the ability to keep track of which user made the changes
+
+Audited has "audits", which contain information about changes to a model, and
+"revisions" which represent the audited model at the time the audit was
+created.
```

Comment:
I wonder if the explanation of what an audit is should go before its mention in the list above, perhaps as a first point?